libs/@hashintel/petrinaut/package.json

```diff
@@ -10,43 +10,54 @@
   "license": "(MIT OR Apache-2.0)",
   "sideEffects": false,
   "type": "module",
-  "main": "src/main.ts",
-  "types": "src/main.ts",
   "scripts": {
-    "dev": "vite --config dev/vite.config.ts",
+    "build": "vite build",
+    "dev": "vite",
     "fix:eslint": "eslint --fix .",
     "lint:eslint": "eslint --report-unused-disable-directives .",
     "lint:tsc": "tsc --noEmit",
     "prepublishOnly": "PACKAGE_DIR=$(pwd) yarn workspace @local/repo-chores exe scripts/prepublish.ts",
-    "postpublish": "PACKAGE_DIR=$(pwd) yarn workspace @local/repo-chores exe scripts/postpublish.ts"
+    "preview": "vite preview",
+    "postpublish": "PACKAGE_DIR=$(pwd) yarn workspace @local/repo-chores exe scripts/postpublish.ts",
+    "test:unit": "vitest"
   },
   "dependencies": {
-    "@emotion/react": "11.14.0",
-    "@fortawesome/free-solid-svg-icons": "6.7.2",
-    "@hashintel/block-design-system": "0.0.5",
+    "@ark-ui/react": "5.26.2",
+    "@babel/core": "7.26.0",
+    "@babel/plugin-transform-typescript": "7.25.9",
+    "@babel/standalone": "7.28.5",
+    "@dnd-kit/core": "6.3.1",
+    "@dnd-kit/sortable": "10.0.0",
+    "@dnd-kit/utilities": "3.2.2",
     "@hashintel/design-system": "0.0.9-canary.2",
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/hashintel/hash/pull/7956/commits/d7f6ce92908ff7c7b1689655c736b5c899143599